tasks/cephfs/fuse_mount.py

```diff
@@ -68,6 +68,13 @@ def mount(self):
         run_cmd.extend(run_cmd_tail)
 
         def list_connections():
+            self.client_remote.run(
```

Comment:
On Tue, May 26, 2015 at 8:00 PM, John Spray <notifications@github.com>
wrote:

> In tasks/cephfs/fuse_mount.py
> <https://github.com/ceph/ceph-qa-suite/pull/444#discussion_r31025977>:
>
> > @@ -68,6 +68,13 @@ def mount(self):
> >          run_cmd.extend(run_cmd_tail)
> >
> >          def list_connections():
> > +            self.client_remote.run(
>
> Do we still even need the mount call here? I don't know the details but I
> always assumed this was something that fuse was supposed to take care of
> for us, and if it isn't present then we can assume there are no fuse
> connections.
>
> If it is necessary for some reason, then the right place to call this is
> probably at the start of mount() rather than in each list_connections()
>
if we mount fusectl at the start of mount, It's possible that the kernel
still does not have fuse.ko inserted.



> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph-qa-suite/pull/444/files#r31025977>.
>